zebra/zserv.c

```diff
@@ -2614,7 +2622,8 @@ static inline void zserv_handle_commands(struct zserv *client,
 					 struct stream *msg,
 					 struct zebra_vrf *zvrf)
 {
-	if (hdr->command > sizeof(zserv_handlers)
+	if (hdr->command > (sizeof(zserv_handlers)
+			    / sizeof(void (*)(ZAPI_HANDLER_ARGS)))
```

Comment:
nit: use `array_size()` here for clarity.

Reply:
Yeah, I though about this after I committed this code and went to bed last night.  I'll clean it up here in a bit.